src/utils/validators/isValidURL.js

```diff
@@ -1,6 +1,6 @@
 export const isValidURL = url => {
     const res = url.match(
-        /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g
+        /(https?:\/\/)(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-zA-Z0-9()]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g
```

Comment:
Since this check is now more strict and requires the https protocol to be present in the url, this function can be replaced by:

```js
export const isValidURL = url => {
  try {
    const url = new URL(string)
    return url.protocol === "https:"
  } catch (error) {
    return false
  }
}
```

Reply:
I like the protocol suggestion, but I was thinking if I should use a regular expression,  a valid URL should be like this:

- must start with either http or https and
- length (2, 256)
- contains domain like .com, .org etc.

Reply:
if more flexibility is needed then yeah a regular expression might make more sense